src/components/partnership/partnership.service.ts

```diff
@@ -53,7 +53,8 @@ export class PartnershipService {
   // helper method for defining properties
   property = (prop: string, value: any) => {
     if (!value) {
-      return [];
+      // return [];
```

Comment:
```suggestion
```

Reply:
I had changed that. Must have not commited.